sass/mixins/_mixins-master.scss

```diff
@@ -87,4 +87,17 @@
 		0 0 1px 0 rgba( $color__text-light, 0.7 );  	// Antialiasing, outer edge.
 }
 
+/* Calculates maximum width for post content */
```

Comment:
If you have some time, it'd be great to take a look around the codebase to see where else this mixin might be useful. We could probably reuse this more frequently if it accepted variables. For example: `postContentMaxWidth( $mobile: 2, $tablet: 8, $desktop: 6)`. That's all a separate PR though. 🙂